app/component/SummaryRow.js

```diff
@@ -259,7 +260,11 @@ const SummaryRow = (
   const startTime = moment(data.startTime);
   const endTime = moment(data.endTime);
   const duration = endTime.diff(startTime);
-
+  const co2value =
+    typeof data.emissionsPerPerson?.co2 === 'number' &&
+    data.emissionsPerPerson?.co2 >= 0
+      ? Math.round(data.emissionsPerPerson?.co2)
+      : -1;
```

Comment:
This code is repeated in a couple of places, maybe add an util for it somewhere, not sure if there is some fitting file somewhere already or do we need a new one.

Reply:
Done